hr_employee_relative/i18n/de.po

```diff
@@ -0,0 +1,190 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* hr_employee_relative
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 14.0\n"
```

Comment:
@pedrobaeza Not sure how the translations work now with Weblate, if these po files need to be here or they should be removed and once the translation is in Weblate they are automatically merged here. 
Should the person doing the migration do the entries in Weblate?

Reply:
Yes, they should stay.